packages/react/src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -152,6 +152,36 @@ const buttonStyles: ComponentSlotStylesPrepared<ButtonProps, ButtonVariables> =
           },
         }),
 
+      ...(p.secondaryAlt && {
+        color: siteVariables.colorScheme.silver.foreground,
+        borderColor: siteVariables.colorScheme.silver.borderColor,
```

Comment:
I don't think `siteVariables.colorScheme.silver.borderColor` value exists... did you mean `siteVariables.colorScheme.silver.border`?

Reply:
That explains why it wasn't working. 🙄